### PR TITLE
feat: quiz category list

### PR DESCRIPTION
**Pull request by Naveenchandar**
* Quiz category list

screencapture-localhost-3001-2022-04-02-18_47_25

### CODE REVIEW

**Review by saicharan-gouru — APPROVED** [review]
-Ui looks really great.
-Well structured code and folders.
-Good that you have added screenshot for reference.
-Good use of context.
-Looks good to merge

**Review by aniketsingh13 — APPROVED** [review]

**Comment by aniketsingh13** [comment]
Ui look good to me

File: src/Components/Card/index.css [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  background-color: var(--white);
+  border-radius: 4px;
+  font-size: 1rem;
+  object-fit: cover;
+  margin: 2rem;
+  padding: 8px;
+}
+
+.card:hover {
+  box-shadow: 10px 10px 5px 0px #000000bf;
```
Comment (maitrakhatri) [inline src/Components/Card/index.css]: use variables for color

**Review by maitrakhatri — APPROVED** [review]
Really good file structure
One suggestion: commit basic react setup in another PR as it does not need to be reviewed.

**Pull request merged**